drivers/sensor/adxl345/Kconfig

```diff
@@ -0,0 +1,10 @@
+# ADXL345, 3-Axis, +/-16g Digital Accelerometer
+
+# Copyright (c) 2020 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config ADXL345
+	bool "ADXL345 Three Axis High-g I2C accelerometer"
```

Comment:
This isn't a "high-g" accelerometer. Please remove the "high-g" part

Reply:
done